opensafely/pull.py

```diff
@@ -104,19 +125,22 @@ def get_local_images():
             "--filter",
             "label=org.opensafely.action",
             "--no-trunc",
-            "--format={{.Repository}}={{.ID}}",
+            "--format={{.Repository}}:{{.Tag}}={{.ID}}",
         ],
         check=True,
         text=True,
         capture_output=True,
     )
-    images = defaultdict(list)
+    images = dict()
     for line in ps.stdout.splitlines():
         if not line.strip():
             continue
 
-        name, sha = line.split("=", 1)
-        images[name].append(sha)
+        line = line.replace("ghcr.io/opensafely-core/", "")
+
+        image, sha = line.split("=", 1)
+        assert image not in images
```

Comment:
Is this likely to happen?  If so can we throw something more meaningful for the user?  I'm thinking a note to contact tech support is likely more than enough!

Reply:
It's not at all likely to happen. I threw it in to check it wasn't, I'll remove it, thanks